JASP-Engine/JASP/po/R-nl.po

```diff
@@ -12,6 +12,418 @@ msgstr ""
+msgstr ""
+
+msgid ""
+"The optimizer failed to find a solution. Probably due to scaling issues "
+"quasi-separation in the data. Try rescaling or removing some of the "
+"predictors."
+msgstr ""
+
+msgid "Effect"
+msgstr "Effecten"
+
+msgid "df"
+msgstr "vg"
+
+msgid "F"
+msgstr "F"
+
+msgid "ChiSq"
+msgstr ""
+
+msgid "p"
+msgstr "p"
+
+msgid "p (bootstrap)"
+msgstr "p (bootstrap)"
+
+msgid "VS-MPR"
+msgstr "VS-MPR"
+
+msgid "VS-MPR (bootstrap)"
+msgstr "VS-MPR (bootstrap)"
+
+msgid "Intercept"
+msgstr "Intercept"
+
+msgid "Warning:"
+msgstr "<em>Waarschuwing:</em>"
+
+msgid "III"
+msgstr "III"
+
+msgid "II"
+msgstr "II"
+
+msgid "Model summary"
+msgstr "Model samenvatting"
+
+msgid "Deviance"
+msgstr "Afwijking"
+
+msgid "Deviance (REML)"
+msgstr "Afwijking (REML)"
+
+msgid "log Lik."
+msgstr "Log aannemelijkheid"
+
+msgid "AIC"
+msgstr "AIC"
+
+msgid "BIC"
+msgstr "BIC"
+
+msgid "Variance/Correlation Estimates"
+msgstr "Variantie/Correlatie Schattingen"
+
+msgid "%s: Variance Estimates"
+msgstr "%s: Variantie Schattingen"
+
+msgid "Term"
+msgstr "Term"
+
+msgid "Std. Deviation"
+msgstr "Standaardafwijking"
+
+msgid "Variance"
+msgstr "Variantie"
+
+msgid "%s: Correlation Estimates"
+msgstr "%s Correlatie Schattingen"
+
+msgid "Residual Variance Estimates"
+msgstr "Residuele Variantie Schattingen"
+
+msgid "Fixed Effects Estimates"
+msgstr "Vaste Effecten Schattingen"
+
+msgid "Estimate"
+msgstr "Schatting"
+
+msgid "SE"
+msgstr "Std. Fout"
+
+msgid "t"
+msgstr "t"
+
+msgid "Plot"
+msgstr "Grafiek"
+
+msgid ""
+"At least one random effects grouping factor needs to be selected in field "
+"'Background data show'."
+msgstr ""
+
+msgid ""
+"Factor levels need to be distinguished by at least one feature. Please, "
+"check one of the 'Distinguish factor levels' options."
+msgstr ""
+
+msgid "Estimated Means and Confidence Intervals"
+msgstr "Geschatte Gemiddelden en Betrouwbaarheidsintervallen"
+
+msgid "Mean"
+msgstr "Gemiddelde"
+
+msgid "Lower"
+msgstr "Onder"
+
+msgid "%s%% CI"
+msgstr "%s%% BI"
+
+msgid "Upper"
+msgstr "Boven"
+
+msgid "Estimated Marginal Means"
+msgstr "Geschatte Marginale Gemiddelden"
+
+msgid "Row"
+msgstr "Rij"
+
+msgid "z"
+msgstr "z"
+
+msgid "Median"
+msgstr "Mediaan"
+
+msgid "%s%% HPD"
+msgstr "%s%% HGI"
+
+msgid "Estimated Trends"
+msgstr "Geschatte Trends"
+
+msgid "%s (slope)"
+msgstr "%s (helling)"
+
+msgid "Contrasts"
+msgstr "Contrasten"
+
+msgid "Odds Ratio"
+msgstr "Odds Ratio"
+
+msgid "Ratio"
+msgstr "Ratio"
+
+msgid "Fit Statistics"
+msgstr "Pasmaten"
+
+msgid "WAIC"
+msgstr "WAIC"
+
+msgid "WAIC (SE)"
+msgstr "WAIC (SE)"
+
+msgid "LOO"
+msgstr "LOO"
+
+msgid "LOO (SE)"
+msgstr "LOO (SE)"
+
+msgid "%s (differences from intercept)"
+msgstr "%s (verschillen t.o.v. intercept)"
+
+msgid "%s (trend)"
+msgstr "s (trend)"
+
+msgid "%s (marginal means)"
+msgstr "%s (marginale gemiddelde)"
+
+msgid "Level"
+msgstr "Niveau"
+
+msgid "R-hat"
+msgstr "R-hat"
```

Comment:
Hier gelden dezelfde comments als in die andere. Met R-dakje (mits we dat daar gaan gebruiken) en gefixeerde/vaste effecten.